companies/123greetings.json

```diff
@@ -1,22 +1,23 @@
 {
-    "slug": "123greetings",
+    "slug": "123greetings2",
```

Comment:
🚫 **[recordlint]** <[recordlint/some_error](https://github.com/datenanfragen/data/issues/1477)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>You made an error here.

abcdef
```suggestion
    "slug": "123greetings",
```